modules/csg/SCsub

```diff
@@ -6,7 +6,43 @@ Import("env_modules")
 
 env_csg = env_modules.Clone()
 
-# Godot source files
+env_csg.Append(CPPDEFINES=("MANIFOLD_PAR", "-1"))
+
+# Thirdparty source files
+
+thirdparty_obj = []
+
+thirdparty_dir = "#thirdparty/manifold/"
+thirdparty_sources = [
+    "src/cross_section/cross_section.cpp",
+    "src/constructors.cpp",
+    "src/polygon.cpp",
+    "src/edge_op.cpp",
+    "src/face_op.cpp",
+    "src/impl.cpp",
+    "src/boolean_result.cpp",
+    "src/boolean3.cpp",
+    "src/manifold.cpp",
+    "src/properties.cpp",
+    "src/smoothing.cpp",
+    "src/sort.cpp",
+    "src/csg_tree.cpp",
+    "src/subdivision.cpp",
+    "src/quickhull.cpp",
```

Comment:
What logic was used for the order of the files? Usually we'd order alphabetically, or in the same order as upstream's buildsystem lists them, which here seems alphabetical too:

https://github.com/elalish/manifold/blob/master/src/CMakeLists.txt#L15-L31

Reply:
It's in random order. How can I sort?

Reply:
Most code editors come with a sort lines function